src/main/resources/api_configuration.json

```diff
@@ -35,6 +35,60 @@
       "method": "GET",
       "pathPattern": "/orders/validate",
       "proxyPath": "/ebsconet/validate"
+    },
+    {
+      "type": "MOSAIC",
+      "method": "GET",
+      "pathPattern": "/mosaic/validate",
+      "proxyPath": "/mosaic/validate"
+    },
+    {
+      "type": "MOSAIC",
+      "method": "POST",
+      "pathPattern": "/mosaic/orders",
+      "proxyPath": "/mosaic/orders"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/orders/order-templates",
+      "proxyPath": "/orders/order-templates?offset=:offset&limit=:limit&query=:query"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/finance/funds",
+      "proxyPath": "/finance/funds?offset=:offset&limit=:limit&query=:query"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/finance/expense-classes",
+      "proxyPath": "/finance/expense-classes?offset=:offset&limit=:limit&query=:query"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/locations",
+      "proxyPath": "/locations?offset=:offset&limit=:limit&query=:query"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/acquisitions-units/units",
+      "proxyPath": "/acquisitions-units/units?offset=:offset&limit=:limit&query=:query"
+    },
+    {
+      "type": "COMMON",
+      "method": "GET",
+      "pathPattern": "/organizations/organizations",
```

Comment:
Is it possible to have just /organizations, the same with acq units, just /acquisitions-units ?